### PR TITLE
Update README.md add more configuration information for registry

### PR DESCRIPTION
**Pull request by sguequierre**
Docs team is doing a sweep of module readmes to make them easier to use and give modules some standard config info (registry usage specific)

* Tried to add some more info notes here, added attributes table, rearranged a bit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,75 @@
...
+| `width_px` | int | Optional | Width in pixels of the images output by this camera. If the OAK-D cannot produce the requested resolution, the component will be configured to the closest resolution to the given height/width. Therefore, the image output size will not always match the input size. Default: `640` |
+| `height_px` | int | Optional | Height in pixels of the images output by this camera. If the OAK-D cannot produce the requested resolution, the component will be configured to the closest resolution to the given height/width. Therefore, the image output size will not always match the input size. Default: `400` |
```
Comment (bazile-clyde) [inline README.md:41]:
```suggestion
| `height_px` | int | Optional | Height in pixels of the images output by this camera. If the OAK-D cannot produce the requested resolution, the component will be configured to the closest resolution to the given height/width. Therefore, the image output size will not always match the input size. Default: `480` |
```

File: README.md [hunk 2]
```diff
@@ -2,9 +2,75 @@
...
+| `frame_rate` | int | Optional | The frame rate the camera will capture images at. Default: `30.0` |
```
Comment (bazile-clyde) [inline README.md:42]:
```suggestion
| `frame_rate` | int | Optional | The frame rate the camera will capture images at. Default: `30` |
```
The type is `int` but this is a float.

**Review by bazile-clyde — APPROVED** [review]
Just a few nits but otherwise, LGTM! Thanks for updating this.

File: README.md (hunk 2 above)
Comment (sguequierre) [reply README.md:42]: Gotcha, thank you!

File: README.md (hunk 1 above)
Comment (sguequierre) [reply README.md:41]: Fixed

**Review by npentrel — APPROVED** [review]

**Pull request merged**